corehq/motech/repeaters/models.py

```diff
@@ -721,6 +726,45 @@ def get_headers(self, repeat_record):
         })
         return headers
 
+    def merge_records(self):
```

Comment:
So if I understand this method correctly, here we're simply deduplicating the repeat records, right? 

The behaviour of this `merge_records` method is different to the `DataSourceRepeater.merge_records` method, which is somewhat surprising to me. 

I like the fact that we're "cleaning up" the repeat records here, but would it not be more suited to do when we create the `CaseRepeater`/`UpdateCaseRepeater`?

Reply:
I'm just thinking maybe we should not use the "merge records" mechanism for this, since if we want to actually chunk the CaseRepeater records in the future for some reason this method would not do that and we'd have to refactor to get it to do that.

Reply:
> Do you mean when we register the repeat record? 

Yes, sorry...conflated the two concepts

Reply:
https://github.com/dimagi/commcare-hq/pull/35734